### ISSUE TITLE
Fetching messages raises undefined method `try' for #<Kafka::FetchedMessage:0x00007feabd1ab120>

### ISSUE THREAD

**Issue by andrewmcnamara** [karafka/karafka#591]
I was trying setup a clean install of Karafka. However when running I try run the Karafka server message consuming fails. Tools like `kafkacat` can consume the messages from kafka - so am pretty confident my setup kafka setup is correct.

As a heads up I have managed to get the clean install working, just needed to add these 2 lines to `karafka.rb`
```
require 'active_support'
require 'active_support/core_ext/object/try'
```

My working example can be found [here](https://github.com/andrewmcnamara/karafka-example)

--------

## Expected behavior

I should be able to run Karafka server standalone.

## Actual behavior

Message consuming failed with error

```
 ERROR -- : Listener fetch loop error: undefined method `try' for #<Kafka::FetchedMessage:0x00007feabd1ab120>
```

## Steps to reproduce the problem
- Setup a Kafka instance 
- Follow these instructions https://github.com/karafka/karafka/wiki/Getting-started  to install a clean setup of Karafka. 
- Create a consumer  e.g.
```
class ExampleConsumer < ApplicationConsumer
  def consume
    Karafka.logger.info "New [User] event: #{params}"
  end
end
```
- Configure the consumer e.g.
```
  consumer_groups.draw do
    consumer_group :my_group do
      topic :topic1 do
        batch_consuming false
        consumer ExampleConsumer
      end
    end
```
- Start the server 

## Your setup details

```
$ [bundle exec] karafka info
Karafka version: 1.3.6
Ruby version: 2.6.6
Ruby-kafka version: 1.1.0
Application client id: example_app
Backend: inline
Batch fetching: true
Batch consuming: false
Boot file: /Users/amcnamara/code/karafka-example/karafka.rb
Environment: development
Kafka seed brokers: ["kafka://127.0.0.1:9094"]
```

**Comment by mensfeld** [comment]
Hey,

First of all, sorry for such a delay. Dunno why but this report went unnoticed :( What is really interesting here, is the fact that AS is ruby-kafka direct dep. Investigating, why it's not a part of the bundle...

**Comment by mensfeld** [comment]
Ok got it: https://github.com/zendesk/ruby-kafka/blob/v1.1.0/lib/kafka/consumer.rb#L319

This is already fixed in the ruby-kafka master. When running from master things work. Here is the fix: https://github.com/zendesk/ruby-kafka/issues/836

I will close this issue and won't fix it, as it will go away soon with 1.1.1 release of ruby-kafka.

Sorry once again for the delay.
